decidim-core/app/packs/stylesheets/decidim/_emoji.scss

```
@@ -0,0 +1,36 @@
+.picmo {
```

Same here, can we rename this to something more generic? I was thinking 
```suggestion
.emoji-picker {
```

But whatever naming you prefer is good for me